1:30 Notes .rtf

```diff
@@ -17,10 +17,10 @@ User Stories:
 \
 \
 
-\fs36 1) When a user navigates to the root path, they will see a listing of the 5 highest rated shoes. \
+\fs36 1) When a user navigates to the root path, they will see a random shoe of the day.\
 \
 \
-2) When a user clicks on a shoe listing, it will direct them to that shoe\'92s individual show page. From there they can view details including a rating across varying qualities, and a price history graph.\
+2) When a user clicks on a shoe listing, it will direct them to that shoe\'92s individual show page. From there they can view details including a rating across varying qualities, written reviews, and a price history graph.\
 	2)a) a user can add their own reviews to pre-existing shoes loaded from API\
 \
 \
```

Comment:
let's remove this notes file from the project, and instead store notes using something like Google Drive